tests/lib/rules/jest/no-vague-titles.js

```diff
@@ -0,0 +1,390 @@
+const {RuleTester} = require('eslint');
+const rule = require('../../../../lib/rules/jest/no-vague-titles');
+
+const ruleTester = new RuleTester();
+
+require('babel-eslint');
+
+const parser = 'babel-eslint';
+
+function errorWithMethod(method, type = 'CallExpression') {
+  return [
+    {
+      type,
+      message: `${method} statement should not contain vague words`,
+    },
+  ];
+}
+ruleTester.run('no-vague-titles', rule, {
+  valid: [
+    {
+      code: `it('foo bar baz')`,
+      parser,
+    },
+    {
+      code: `describe('foo bar baz')`,
+      parser,
+    },
+    {
+      code: `test('foo bar baz')`,
```

Comment:
You should also test non-test functions called with an argument that contains the bad words